pymatgen/symmetry/groups.py

```diff
@@ -223,13 +236,13 @@ def __init__(self, int_symbol: str) -> None:
         for spg in SpaceGroup.SYMM_OPS:
             if int_symbol in [spg["hermann_mauguin"], spg["universal_h_m"]]:
                 ops = [SymmOp.from_xyz_str(s) for s in spg["symops"]]
-                self.symbol = re.sub(r":", "", re.sub(r" ", "", spg["universal_h_m"]))
+                self.symbol = spg["hermann_mauguin"]
                 if int_symbol in SpaceGroup.sg_encoding:
                     self.full_symbol = SpaceGroup.sg_encoding[int_symbol]["full_symbol"]
                     self.point_group = SpaceGroup.sg_encoding[int_symbol]["point_group"]
                 else:
-                    self.full_symbol = re.sub(r" ", "", spg["universal_h_m"])
-                    self.point_group = spg["schoenflies"]
+                    self.full_symbol = spg["hermann_mauguin"]  # TODO full symbol not available from spg->maybe not add?
+                    self.point_group = spg["schoenflies"].split("^")[0]  # TODO map onto Hermann Mauguin notation?
```

Comment:
let's come to a decision here before merging this PR. what would be the benefits of mapping onto Hermann Mauguin notation?

Reply:
The information is the same, but in these cases we'd have different notations for space group (Hermann Mauguin) and point group (Schoenflies) within one object. Also, it might cause issues if one wants to compare the point groups for a larger dataset (and the same point group could be there in different representations).

In order to map it, we would either need a database or we could, in principle, extract the point group in Hermann Mauguin notation from the space group symbol (by removing the centering and translation parts of symmetry elements) - this would require additional testing, though.

Reply:
Maybe there could be a PointGroup class method from_spacegroup() that does exactly this?

Reply:
that sounds like a good idea! would you prefer to do that in a separate PR (or just open an issue for it)?

Reply:
Great, i'd prefer to just open an issue for it and add it in this PR - will do both tomorrow (:

Reply:
Added the new class method and also added this info directly to the `symm_ops.json` database so it does not have to be called every time when initializing a `SpaceGroup` object. The changes to `symm_ops.json` and `symm_data.json` are documented in `dev_scripts.update_space_group_data.py` - was not sure if this is the correct place, though!

Reply:
thank you! this is great. it's super valuable to leave such a thorough paper trial for people to understand what happened later